lib/dialects/mssql/query.js

```diff
@@ -211,6 +213,20 @@ class Query extends AbstractQuery {
     });
   }
 
+  handleShowConstraintsQuery(data) {
+    data.splice(0, 1);
+    //Convert snake_case keys to camelCase as its generated by stored procedure
+    for (const constraint of data) {
+      for (const key in constraint) {
+        const camelKey = _.camelCase(key);
+        constraint[camelKey] = constraint[key];
+        delete constraint[key];
```

Comment:
@felixfbecker  Setting it to undefined keeps both snake & camel cased keys to exist, though one of them would be undefined. I think its better to create a new object. Is this alright?
```javascript
handleShowConstraintsQuery(data) {
    data.splice(0, 1);
    //Convert snake_case keys to camelCase as its generated by stored procedure
    const constraints = data.map(result => {
      const constraint = {};
      for (const key in result) {
        const camelKey = _.camelCase(key);
        constraint[camelKey] = result[key];
      }
      return constraint;
    });

    return constraints;
  }
```